src/css/tabs/led_strip.less

```diff
@@ -150,14 +150,14 @@
 			margin-left: -9px;
 		}
 	}
-	.gPoint.function-n {
-		.overlay-n {
```

Comment:
Needed for backward compatibility

Reply:
I think we don't need this, the configurator only compatible from bf 4.0+ and I just checked and bf 4.0 doesn't contain this overlay.
It's even hidden in the process_html function in `led_strip.js` line 88 and only occurs once in the file, there.
So I think it's not supported now anyway.